src/components/role-manager.ts

```diff
@@ -5,14 +5,12 @@ import { unwrap } from "../utils/misc.js";
 import { M } from "../utils/debugging-and-logging.js";
 import { BotComponent } from "../bot-component.js";
 import { CommandSetBuilder } from "../command-abstractions/command-set-builder.js";
-import { skill_roles_order, skill_roles_order_id, Wheatley } from "../wheatley.js";
 import { set_interval } from "../utils/node.js";
 import { build_description } from "../utils/strings.js";
 
-type user_role_entry = {
+export type user_role_entry = {
     user_id: string;
     roles: string[];
-    last_known_skill_role: string | null;
 };
```

Comment:
This is a schema so changing it like this isn't great. I'll have to think about how to do it better.